components/admin/requests/guardian-information/Form.tsx

```diff
@@ -153,15 +156,31 @@ export default function GuardianInformationForm({
             <Text as="h3" textStyle="heading" paddingBottom="20px">
               {'Upload POA File'}
             </Text>
-            {/* TODO: Implement the file upload functionality using the fileList prop */}
+            {/* TODO: explicitly prevent non pdfs or files > 5Mb? */}
             <Text color="text.secondary">
               {
                 'Only ONE file can be added. Files must be .pdf and can be a maximum of 5MB in size.'
               }{' '}
             </Text>
-            <Button variant="solid" marginTop="15px" leftIcon={<AttachmentIcon />}>
-              {'Click to add attachement'}
-            </Button>
+            {file && (
+              <>
+                <Text as="h4" textStyle="button-semibold" mt="24px">
+                  Current File
+                </Text>
+                <Box mt="8px">
+                  <Link
+                    href={URL.createObjectURL(file)}
```

Comment:
Just wondering - does this actually create a clickable link to open the file? What does this line do?

Reply:
Yeah, it makes a temporary url so when you click the link it will download the file. 
https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL